Kickstarter-iOS/Views/Controllers/PledgeContinueViewController.swift

```diff
@@ -5,7 +5,7 @@ import Prelude
 final class PledgeContinueViewController: UIViewController {
   // MARK: - Properties
 
-  private let continueButton = MultiLineButton(type: .custom)
+  private let continueButton = UIButton(type: .custom)
```

Comment:
We no longer need to use `MultiLineButton` here...in fact since we've previously added trimming we haven't been using it for a while (so this is just a cleanup).

Reply:
I see we're only using this in one place, not time to remove `MultiLineButton` yet?